projects/1.20/assets/bygone-nether/bygonenether/lang/zh_cn.json

```
@@ -0,0 +1,111 @@
+{
+	"item.bygonenether.gilded_netherite_helmet": "镀金下界合金头盔",
+	"item.bygonenether.gilded_netherite_chestplate": "镀金下界合金胸甲",
+	"item.bygonenether.gilded_netherite_leggings": "镀金下界合金护腿",
+	"item.bygonenether.gilded_netherite_boots": "镀金下界合金靴子",
+
+	"item.bygonenether.cobbled_blackstone": "黑石圆石",
+	"item.bygonenether.cracked_withered_blackstone": "裂纹凋零黑石",
+	"item.bygonenether.cracked_withered_blackstone_stairs": "裂纹凋零黑石楼梯",
+	"item.bygonenether.cracked_withered_blackstone_slab": "裂纹凋零黑石台阶",
+	"item.bygonenether.chiseled_withered_blackstone": "錾制凋零黑石",
+	"item.bygonenether.withered_blackstone": "凋零黑石",
+	"item.bygonenether.withered_blackstone_stairs": "凋零黑石楼梯",
+	"item.bygonenether.withered_blackstone_slab": "凋零台阶",
+	"item.bygonenether.withered_debris": "凋零残骸",
+	"item.bygonenether.soul_stone": "灵魂石",
+	"item.bygonenether.warped_nether_bricks": "诡异下界砖",
+	"item.bygonenether.chiseled_warped_nether_bricks": "錾制诡异下界砖",
+	"item.bygonenether.warped_nether_brick_stairs": "诡异下界砖楼梯",
+	"item.bygonenether.warped_nether_brick_slab": "诡异下界砖台阶",
+	"item.bygonenether.withered_basalt": "凋零玄武岩",
+	"item.bygonenether.withered_coal_block": "凋零煤炭块",
+	"item.bygonenether.withered_quartz_block": "凋零石英块",
+
+	"item.bygonenether.piglin_prisoner_spawn_egg": "猪灵囚犯刷怪蛋",
+	"item.bygonenether.piglin_hunter_spawn_egg": "猪灵猎人刷怪蛋",
+	"item.bygonenether.wex_spawn_egg": "凋灵恼鬼刷怪蛋",
+	"item.bygonenether.warped_enderman_spawn_egg": "诡异末影人刷怪蛋",
+	"item.bygonenether.wraither_spawn_egg": "凋零死士刷怪蛋",
+	"item.bygonenether.corpor_spawn_egg": "凋零蛮兵刷怪蛋",
+	"item.bygonenether.wither_skeleton_knight_spawn_egg": "凋零骷髅骑士刷怪蛋",
+	"item.bygonenether.wither_skeleton_horse_spawn_egg": "凋灵骷髅马刷怪蛋",
+
+	"item.bygonenether.wither_waltz_music_disc": "音乐唱片",
+	"item.bygonenether.wither_waltz_music_disc.desc": "izofar - 凋灵华尔兹",
+
+	"item.bygonenether.warped_ender_pearl": "扭曲的末影珍珠",
+	"item.bygonenether.netherite_bell": "下界合金钟",
+	"item.bygonenether.gilded_netherite_shield": "镀金合金盾",

```

```suggestion
	"item.bygonenether.gilded_netherite_shield": "镶金下界合金盾",
```
这个`Gilded Netherite`是怎么合成的